Templates/AppSource App/.github/workflows/UpdateGitHubGoSystemFiles.yaml

```diff
@@ -74,18 +74,44 @@ jobs:
 
       - name: Calculate Input
         env:
-          directCommit: '${{ github.event.inputs.directCommit }}'
+          directCommit: ${{ github.event.inputs.directCommit }}
           downloadLatest: ${{ github.event.inputs.downloadLatest }}
           eventName: ${{ github.event_name }}
+          GhTokenWorkflow: ${{ fromJson(steps.ReadSecrets.outputs.Secrets).ghTokenWorkflow }}
```

Comment:
`ghTokenWorkflow` is already set as env variable by `ReadSecrets`, isn't it?

Reply:
Secrets are no longer added as ENV vars - only in GITHUB_OUTPUT